<launch>
    <!--*******************************      Sensor Processor       *******************************-->

    <!--*******************************         Arguments           *******************************-->

    <!--*******************************           Nodes             *******************************-->

    <node pkg="lara_rgbd" type="state_estimator" name="state_estimator" output="screen" />

</launch>
